pkg/transforms/vapbinding.go

```diff
@@ -51,6 +89,105 @@ func (v VapBindingResource) BuildNode() Node {
 
 // BuildEdges construct the edges for VapBindingResource
 func (v VapBindingResource) BuildEdges(ns NodeStore) []Edge {
-	// no op for now to implement interface
-	return []Edge{}
+	policyName, ok := v.node.Properties["policyName"].(string)
+	if !ok || policyName == "" {
+		return []Edge{}
+	}
+
+	nodeInfo := NodeInfo{
+		Name:      v.node.Properties["name"].(string),
+		NameSpace: "_NONE",
+		UID:       v.node.UID,
+		EdgeType:  "attachedTo",
+		Kind:      v.node.Properties["kind"].(string),
+	}
+
+	propSet := map[string]struct{}{policyName: {}}
+
+	edges := edgesByDestinationName(propSet, "ValidatingAdmissionPolicy", nodeInfo, ns, []string{})
+
+	if v.paramRef == nil {
+		return edges
+	}
+
+	policy, ok := ns.ByKindNamespaceName["ValidatingAdmissionPolicy"]["_NONE"][policyName]
+	if !ok {
+		return edges
+	}
+
+	paramKind := policy.Metadata["paramKind_kind"]
+	if paramKind == "" {
+		return edges
+	}
+
+	paramApiVersion := policy.Metadata["paramKind_apiVersion"]
+	if paramApiVersion == "" {
+		return edges
+	}
+
+	paramGV, err := schema.ParseGroupVersion(paramApiVersion)
+	if err != nil {
+		return edges
+	}
+
+	namespaceToName := ns.ByKindNamespaceName[paramKind]
+
+	if v.paramRef.Namespace != "" {
```

Comment:
If paramRef.namesapce is empty or '' (clusterscope), 
```
ns.ByKindNamespaceName[paramKind]["_NONE"] need?
```

Reply:
It's just an optimization when the namespace is specified. If it's not specified then all namespaces, including cluster scoped objects in `_NONE` are searched.